__tests__/app.test.js

```diff
@@ -21,8 +22,101 @@ test('Get array with topic objects', () => {
     })
 
 });
-    
+});
+
+describe('/api/articles/:article_id', () => {
+    test('responds with an article object of article id 1', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((res) => {
+        expect(res.body.article).toHaveLength(1)
```

Comment:
Your test description says you are responding with an object but these assertions are for an array of length 1. 

Consider adjusting them to be just for the object as you did in the PATCH request.

Reply:
OH Thanks paul